### DISCUSSION TITLE
How to remove cloud link option.?

### DISCUSSION THREAD

**Discussion by bosskie** [anasty17/mirror-leech-telegram-bot#1738] [Q&A]
![image](https://github.com/user-attachments/assets/ed02c9e0-c77b-4efd-b01d-63c69ede346a)
Anyone know how to remove cloud link option ?
I just want to have index link option.

Thanks.

**Comment by anasty17** [comment]
U need to edit the code

**Reply by bosskie** [reply]
Can you specify which file in the repo to edit this out..? 

Thanks @anasty17

**Reply by anasty17** [reply]
Sorry can't help in edits
